nixos/modules/services/mail/nullmailer.nix

```diff
@@ -212,6 +212,9 @@ with lib;
 
     systemd.tmpfiles.rules = [
       "d /var/spool/nullmailer - ${cfg.user} - - -"
```

Comment:
Shouldn't this also be 750?

Reply:
I think it could, but it's not necessary. Even if the permission is the default 755, it doesn't give access to anything sensitive. Besides the subdirectories, there is only `/var/spool/nullmailer/trigger`, with permission 660. I didn't want to change the permissions here, because somebody may depend on it somehow.

Reply:
It turns out if we want to keep systemd hardening options and do not rely on setuid binaries, this needs to be widened to 770 and group nullmailer. Creating a PR for that later.